app/_redirects

```diff
@@ -647,68 +647,16 @@
-/mesh/:version/networking/networking/ /mesh/:version/production/deployment/networking/
-/mesh/:version/explore/zone-ingress/  /mesh/:version/production/cp-deployment/zone-ingress/
-/mesh/:version/security/zoneproxy-auth/ /mesh/:version/production/cp-deployment/zoneproxy-auth/
-/mesh/:version/explore/zoneegress/  /mesh/:version/production/cp-deployment/zoneegress/
-/mesh/:version/deployments/stand-alone/#usage /mesh/:version/production/stand-alone/
-/mesh/:version/policies/mesh/ /mesh/:version/production/mesh/
-/mesh/:version/explore/dpp/ /mesh/:version/production/dp-config/dpp/
-/mesh/:version/explore/dpp-on-kubernetes/ /mesh/:version/production/dp-config/dpp-on-kubernetes/
-/mesh/:version/explore/dpp-on-universal/  /mesh/:version/production/dp-config/dpp-on-universal/
-/mesh/:version/networking/cni/  /mesh/:version/production/dp-config/cni/
-/mesh/:version/networking/transparent-proxying/ /mesh/:version/production/dp-config/transparent-proxying/
-/mesh/:version/networking/ipv6/ /mesh/:version/production/dp-config/ipv6/
-/mesh/:version/security/secrets/  /mesh/:version/production/secure-deployment/secrets/
-/mesh/:version/security/api-access-control/ /mesh/:version/production/secure-deployment/api-access-control/
-/mesh/:version/security/api-server-auth/  /mesh/:version/production/secure-deployment/api-server-auth/
-/mesh/:version/security/dp-auth/  /mesh/:version/production/secure-deployment/dp-auth/
-/mesh/:version/security/dp-membership/  /mesh/:version/production/secure-deployment/dp-membership/
-/mesh/:version/security/certificates/ /mesh/:version/production/secure-deployment/certificates/
-/mesh/:version/explore/gui/ /mesh/:version/production/gui/
-/mesh/:version/documentation/upgrades/  /mesh/:version/production/upgrades-tuning/upgrades/
-/mesh/:version/documentation/fine-tuning/ /mesh/:version/production/upgrades-tuning/fine-tuning/
-/mesh/:version/installation/* /mesh/:version/production/install-kumactl/
-/mesh/:version/install/* /mesh/:version/production/install-kumactl/
-
-
-# Kuma IA Reference section
-
-/docs/:version/policies/proxy-template/ /docs/:version/reference/proxy-template/
-
-# Redirect missing policies to features (especially useful to make doc links work)
-/mesh/:version/policies/* /mesh/:version/features/:splat
-
-
-## Introduction
-/docs/:version/introduction/what-is-kuma/ /docs/:version/introduction/overview-of-kuma/ 302
-/docs/:version/introduction/what-is-a-service-mesh/ /docs/:version/introduction/what-is-a-service-mesh/ 302
-/docs/:version/introduction/deployments/  /docs/:version/production/deployment/ 302
+# Make GUI docs link work
+/mesh/konnect/* /mesh/latest/:splat
+/mesh/:version/installation/windows   /mesh/:version/features/windows
+/mesh/:version/policies/meshopa   /mesh/:version/features/meshopa
+/mesh/:version/policies/meshglobalratelimit   /mesh/:version/features/meshglobalratelimit
+/mesh/changelog /mesh/latest/introduction/changelog
 
```

Comment:
Single comment here instead of inline because it might be easier:
 
We should keep these: (Note that the destination URL changed the first part of the segment from `/docs/` to `/mesh/`) 
* /docs/:version/introduction/what-is-kuma/ /mesh/:version/introduction/overview-of-kong-mesh/ 302
* /docs/:version/introduction/what-is-a-service-mesh/ /mesh/:version/introduction/what-is-a-service-mesh/ 302
* /docs/:version/introduction/deployments/  /mesh/:version/production/deployment/ 302
* /docs/:version/policies/proxy-template/ /mesh/:version/reference/proxy-template/

We need to keep:
* /mesh/:version/features/windows/     /mesh/:version/installation/windows/ for versions  1.6.x and 1.5.x, anything older that returns 404 even on prod, neither page exists
* /mesh/:version/introduction/what-is-kong-mesh   /mesh/:version/
* /install/:version/  /mesh/:version/install/
* /mesh/:version/policies/protocol-support-in-kuma/  /mesh/:version/policies/protocol-support-in-kong-mesh/
* /mesh/:version/plan-and-deploy/license    /mesh/:version/production/cp-deployment/license/
* / mesh/:version/introduction/deployments/  /mesh/:version/production/deployment/ 			
* /mesh/:version/deployments/stand-alone/ /mesh/:version/production/deployment/stand-alone/
* /mesh/:version/deployments/multi-zone/  /mesh/:version/production/deployment/multi-zone/
* /mesh/:version/deployments/multi-zone/#usage  /mesh/:version/production/cp-deployment/multi-zone/
* /mesh/2.1.x/deployments/multi-zone/#usage /mesh/2.1.x/production/cp-deployment/multi-zone/
* /mesh/:version/explore/zone-ingress/  /mesh/:version/production/cp-deployment/zone-ingress/ 
* /mesh/:version/security/zoneproxy-auth/ /mesh/:version/production/cp-deployment/zoneproxy-auth/
* /mesh/:version/explore/zoneegress/  /mesh/:version/production/cp-deployment/zoneegress/
* /mesh/:version/policies/mesh/ /mesh/:version/production/mesh/
* /mesh/:version/explore/dpp-on-kubernetes/ /mesh/:version/production/dp-config/dpp-on-kubernetes/
* /mesh/:version/explore/dpp-on-universal/  /mesh/:version/production/dp-config/dpp-on-universal/
* /mesh/:version/networking/cni/  /mesh/:version/production/dp-config/cni/
* /mesh/:version/networking/transparent-proxying/ /mesh/:version/production/dp-config/transparent-proxying/
* /mesh/:version/networking/ipv6/ /mesh/:version/production/dp-config/ipv6/
* /mesh/:version/security/secrets/  /mesh/:version/production/secure-deployment/secrets/
* /mesh/:version/security/api-access-control/ /mesh/:version/production/secure-deployment/api-access-control/
* /mesh/:version/security/api-server-auth/  /mesh/:version/production/secure-deployment/api-server-auth/
* /mesh/:version/security/dp-auth/  /mesh/:version/production/secure-deployment/dp-auth/
* /mesh/:version/security/dp-membership/  /mesh/:version/production/secure-deployment/dp-membership/
* /mesh/:version/security/certificates/ /mesh/:version/production/secure-deployment/certificates/
* /mesh/:version/explore/gui/ /mesh/:version/production/gui/
* /mesh/:version/documentation/upgrades/  /mesh/:version/production/upgrades-tuning/upgrades/
* /mesh/:version/documentation/fine-tuning/ /mesh/:version/production/upgrades-tuning/fine-tuning/
* /mesh/:version/installation/* /mesh/:version/production/install-kumactl/
* /mesh/:version/install/* /mesh/:version/production/install-kumactl/
* /mesh/:version/networking/networking/ /mesh/:version/production/deployment/networking/ -> destination URL 404ing because it’s no longer present in the nav file
* /mesh/:version/deployments/stand-alone/#usage /mesh/:version/production/stand-alone/ -> destination URL 404ing
*/mesh/:version/explore/dpp/ /mesh/:version/production/dp-config/dpp/ -> destination URL 404ing 


We can safely remove:
* /mesh/1.0.x/features/   /mesh/1.0.x/features/vault
* /mesh/1.1.x/features/   /mesh/1.1.x/features/vault
* /mesh/:version/features/rbac    /mesh/:version/production/secure-deployment/
* /mesh/:version/features/fips-support  /mesh/:version/production/secure-deployment/fips-support/
* /mesh/:version/features/meshopa   /mesh/:version/policies/meshopa -> for versions <= 2.0.x returns an infinite redirect error we dont need it for >= 2.1.x what should we do with old versions?
* /mesh/:version/features/meshglobalratelimit   /mesh/:version/policies/meshglobalratelimit -> for versions <= 2.0.x returns an infinite redirect error we dont need it for >= 2.1.x what should we do with old versions?
* /mesh/2.1.x/deployments/stand-alone/#usage  /mesh/2.1.x/production/cp-deployment/stand-alone/

Reply:
So these:

```
/docs/:version/introduction/what-is-kuma/ /mesh/:version/introduction/overview-of-kong-mesh/ 302
/docs/:version/introduction/what-is-a-service-mesh/ /mesh/:version/introduction/what-is-a-service-mesh/ 302
/docs/:version/introduction/deployments/ /mesh/:version/production/deployment/ 302
/docs/:version/policies/proxy-template/ /mesh/:version/reference/proxy-template/
```

Should always be handled in `_common_redirects` so that we don't have to play differently for Kuma and Kong-Mesh. Also there's nowhere where we link to `what-is-kuma` in the Kong-Mesh docs as the page doesn't exist.

For the others:

1. /mesh/:version/features/windows/ /mesh/:version/installation/windows/ for versions 1.6.x and 1.5.x, anything older that returns 404 even on prod, neither page exists -- This is take care of above
2. /mesh/:version/introduction/what-is-kong-mesh /mesh/:version/ -- Do we have data from google analytics pointing that this URL has ever been visited? (My gutt would be that it hasn't)
3. /install/:version/ /mesh/:version/install/ --> While this existed it's clearly not our right to take the whole `/install` path. So I think this should be removed anyway
4. /mesh/:version/policies/protocol-support-in-kuma/ /mesh/:version/policies/protocol-support-in-kong-mesh/ Correctly fixed the url and title plugin so that this works well once this is merged
5. For the rest I believe they are either in _common_redirects or just redirects for pages that are long gone.

In the future we should rely stay away from _redirects for moving pages and only rely in changing the `url` in the nav bar. This makes these things incredibly easier to maintain.